changelogs/fragments/remove_deprecated.yaml

```diff
@@ -0,0 +1,5 @@
+---
+major_changes:
```

Comment:
Can you please add a removed features section such as this: https://github.com/ansible-collections/cisco.nxos/pull/836/files#diff-044354a2e5989396b9168d5dde63bd5196447bfcf661637e7433555b532c50c6R4?